data/json/recipes/tools/tools_electronic.json

```
@@ -26,7 +26,7 @@
     "decomp_learn": 2,
     "autolearn": [ [ "mechanics", 3 ], [ "electronics", 2 ] ],
     "book_learn": [ [ "manual_mechanics", 3 ], [ "textbook_mechanics", 2 ], [ "welding_book", 1 ] ],
-    "proficiencies": [ { "proficiency": "prof_welding_basic" }, { "proficiency": "prof_welding" } ],
+    "proficiencies": [ { "proficiency": "prof_welding_basic" }, { "proficiency": "prof_welding", "time_multiplier": 1.1, "fail_multiplier": 1.1, "learning_time_multiplier": 0.25 } ],
```

```suggestion
    "proficiencies": [
      { "proficiency": "prof_welding_basic" },
      {
        "proficiency": "prof_welding",
        "time_multiplier": 1.1,
        "fail_multiplier": 1.1,
        "learning_time_multiplier": 0.25
      }
    ],
```

The line was too long, I just threw this into the web linter (http://dev.narc.ro/cataclysm/format.html)